tests/orderly/test_read.py

```diff
@@ -0,0 +1,54 @@
+import ast
+import re
+import sys
+
+import pytest
+from orderly.read import _read_py, orderly_read
```

Comment:
pycharm warns about _read_py being a protected member of a module - but the tests all work fine so it doesn't seem a problem.

Reply:
yeah, just pulling out something to test here